engine/particle/src/particle.cpp

```diff
@@ -45,18 +45,21 @@ namespace dmParticle
     const static Vector3 VORTEX_LOCAL_START_DIR = -Vector3::xAxis();
 
     /// Config key to use for tweaking maximum number of instances in a context.
-    const char* MAX_INSTANCE_COUNT_KEY          = "particle_fx.max_count";
+    const char* MAX_INSTANCE_COUNT_KEY = "particle_fx.max_count";
     /// Config key to use for tweaking maximum number of emitters in a context.
-    const char* MAX_EMITTER_COUNT_KEY          = "particle_fx.max_emitter_count";
+    const char* MAX_EMITTER_COUNT_KEY  = "particle_fx.max_emitter_count";
     /// Config key to use for tweaking the total maximum number of particles in a context.
-    const char* MAX_PARTICLE_COUNT_KEY          = "particle_fx.max_particle_count";
+    const char* MAX_PARTICLE_COUNT_KEY = "particle_fx.max_particle_count";
 
     /// Used for degree to radian conversion
     const float DEG_RAD = (float) (M_PI / 180.0);
 
     /// Simulate motion blur at 60 fps with a 180 deg shutter
     const static float STRETCH_SCALING = (1.0f/60.0f) * 0.5f;
 
+    // JG: Would be nice to share these somewhere more locally, like a constant hash directory in dlib or something..
+    static const dmhash_t VERTEX_STREAM_COLOR = dmHashString64("color");
```

Comment:
I need to have this value somewhere since we need to distinguish between a "built-in" color stream from the engine and a custom stream. Or in short, we can't just rely on the semantic type to output a color, we need the name of the stream as well. (See how we write vertices later on..)